ui/admin/tests/helpers/selectors.js

```diff
@@ -39,3 +39,7 @@ export const DIALOG_UNSAVED_CHANGES_DISCARD =
   '.rose-dialog-footer button:first-child';
 export const DIALOG_UNSAVED_CHANGES_CANCEL =
   '.rose-dialog-footer button:last-child';
+
+export const MODAL_WARNING = '[data-test-modal]';
```

Comment:
As mentioned above, I will suggest to not use `data-test-*` selectors for this use case.

In this case, unless there is an specific need I am missing I will encourauge for use standard selector. I.e:

```
export const MODAL_WARNING = 'dialog';
export const MODAL_WARNING_CONFIRM_BTN = 'dialog .hds-modal__footer button:first-child';
export const MODAL_WARNING_CANCEL_BTN = 'dialog .hds-modal__footer button:last-child';
```